File: server/Models/Clan.cs
namespace PaceUpServer.Models
{
    public class Clan
    {
        public string Id { get; set; } = "";
        public string Name { get; set; } = "";
        public string Description { get; set; } = "";
        public int TotalXp { get; set; } = 0;
        public int MemberCount { get; set; } = 0;
        public int MaxMembers { get; set; } = 20;
        public string LeaderId { get; set; } = "";
    }
}

File: app/src/main/java/com/example/paceup/model/Clan.kt
package com.example.paceup.model

data class Clan(
    val id: String = "",
    val name: String = "",
    val description: String = "",
    val totalXp: Int = 0,
    val memberCount: Int = 0,
    val maxMembers: Int = 20,
    val leaderId: String = ""
)